ocs_ci/ocs/ui/page_objects/object_storage.py

```diff
@@ -83,9 +84,15 @@ def nav_object_buckets_claims_tab(self):
 
         return ObjectBucketClaimsTab()
 
-    def navigate_namespace_store_tab(self):
-        logger.info("Navigate to Namespace Store Tab")
-        self.do_click(locator=self.validation_loc["namespacestore_page"])
-        from ocs_ci.ocs.ui.page_objects.namespace_store_tab import NameSpaceStoreTab
+    def select_project(self, cluster_namespace):
+        """
+        Helper function to select openshift-storage project
```

Comment:
If I understand correctly, this method could be used to select any project, not just openshift-storage, so the docstring and the logging should reflect that.